### ISSUE TITLE
sprite designer [-] [+] buttons 

### ISSUE THREAD

**Issue by rodrigo1593** [ArcadeMakerSources/ArcadeMaker-Demo#10]
it works fine with larger sprites but when i tried with 4 x 8 sprite when i press them the sprite drawing area just disappears

**Comment by ArcadeMakerSources** [comment]
@rodrigo1593 Yeah... I would edit the sprites in GameMaker and just import them to arcade maker
